packages/react/src/elements/components/GlobalNav/TopNav/Notifications.js

```diff
@@ -131,14 +131,41 @@ class Notifications extends Component {
 }
 
 Notifications.propTypes = {
+  /**
+   * Opens the notifications flyout
```

Comment:
Interesting to see that react-docgen expects the description to be wrapped by a multi-line comment block (`/* */`). I initially tried `//` and my description was not recognized.